### DISCUSSION TITLE
Can this work or be made to work with ksh?

### DISCUSSION THREAD

**Discussion by myfirstnameispaul** [scop/bash-completion#640] [Q&A]
I freely recognize this is a dumb-guy question, but for my basic usage of ksh this level of completion is the only thing I miss when using OpenBSD default of ksh.

Is it possible to use this with ksh? I don't mean through some herculean rewrite.

I do see it in packages, but I also see bash in packages, so I assumed this is to support bash.

**Comment by scop** [comment]
I don't think bash-completion works with ksh at all. But then again I know very little about ksh.
